### PR TITLE
Mp factors scenes styles

### PR DESCRIPTION
**Pull request by prechter**
Styling for mobile views on these two sections since they're similar. Currently working on tablet/desktop but will save for another PR. Surge link >>>> [HERE](http://dear-watch.surge.sh/)

Key Factors section (Figma):
screen shot 2019-02-12 at 1 54 44 pm

Key Factors section (me
screen shot 2019-02-12 at 2 09 10 pm
):

Behind the Scenes section (Figma):
screen shot 2019-02-12 at 1 55 03 pm

Behind the Scenes section (me):
screen shot 2019-02-12 at 2 09 26 pm

### CODE REVIEW

File: src/stylesheets/blocks/_behind-the-scenes.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  .-heart {
+    top: 49%;
+    left: 62%;
+  }
+
+  .-link {
+    top: 66%;
+    left: 5%;
```
Comment (benjtinsley) [inline src/stylesheets/blocks/_behind-the-scenes.scss:39]: do these percentage based positions stay where they are supposed?

**Review by benjtinsley — APPROVED** [review]
looks good to me.

one thing id recommend using is [surge](https://surge.sh/) to spin up a staging site of the branches you are working on since this is a static site. 

all you would do is install surge on your machine with 
```bash
npm install -g surge
```
then when you are ready for folks to review your code, run the `yarn blendid --build` command to compile your assets for production. then run `surge` and it will give you some prompts. make sure you point the directory to `public/` or wherever blendid compiles your production ready assets. you can give it a unique url, too!

if you are interested in doing this and are having trouble, you can send me a DM and ill help out.

File: src/stylesheets/blocks/_behind-the-scenes.scss (hunk 1 above)
Comment (prechter) [reply src/stylesheets/blocks/_behind-the-scenes.scss:39]: i was curious about this too. originally they didn't when i had `width: auto` on the parent element, but after specifying a width they stay put. i'm using `rem` values to specify position on another stylesheet so it's a little inconsistent.

File: src/stylesheets/utility/_icons.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
+.tag-list__icon,
+.tag-list__btn {
+  align-items: center;
+  background-color: color(yellow);
+  border-radius: 50%;
+  border: solid rem(4.5) color(white);
+  box-shadow: 0 0 0 rem(6) #FFFFFF90;
```
Comment (jeremyfrank) [inline src/stylesheets/utility/_icons.scss:7]: Just a heads up that this 8 digit hex code format is supported in Firefox & Chrome, but not IE or Edge. The alternative with wider support is `rgba()`.

**Review by jeremyfrank — APPROVED** [review]
Mobile looks good. I'd add a quick little snippet to restrict the widths of the chat images though (not related to this work), just to make your mobile testing easier:

```
img {
  max-width: 100%;
}
```

**Pull request merged**